project/BinaryIncompatibilities.scala

```diff
@@ -6,6 +6,40 @@ object BinaryIncompatibilities {
   )
 
   val Tools = Seq(
+      ProblemFilters.exclude[MissingMethodProblem](
+        "org.scalajs.core.tools.linker.backend.emitter.ScalaJSClassEmitter.this"),
```

Comment:
You should add comment explaining why it is OK to ignore those binary compat errors, e.g.,
```scala
// private[emitter], not an issue
```
(or whatever fits)